util/Migrator/DbScripts/2020-06-23_00_OrgIdentifier.sql

```diff
@@ -0,0 +1,18 @@
+﻿IF COL_LENGTH('[dbo].[Organization]', 'Identifier') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[Organization]
+    ADD
+        [Identifier] NVARCHAR (50) NULL
+END
+GO
+
+IF NOT EXISTS (
+    SELECT * FROM sys.indexes  WHERE [Name]='IX_Organization_Identifier'
+    AND object_id = OBJECT_ID('[dbo].[Organization]')
+)
+BEGIN
+    CREATE NONCLUSTERED INDEX [IX_Organization_Identifier]
```

Comment:
@kspearrin , do you think we would want to make this a `UNIQUE` index with a `NOT NULL` filter? I wonder how we're going to validate early vs. late in the pipeline when creating an identifier for an organization to ensure it's unique.

Reply:
We'll need to do a check when a user is setting it to make sure it isn't already taken. We should enforce the unique constraint in the database, too.

Reply:
@kspearrin From what I can see, adding `UNIQUE` to nvarchar places byte-size restrictions on the content length, but I'm not sure if that's what you're really asking.  Am I in the wrong rabbit-hole?

Reply:
Chad explained - we're good